tests/RAGchain/utils/linker/test_base_linker.py

```diff
@@ -1,8 +1,9 @@
 import pytest
+from uuid import uuid4
 
 from RAGchain.utils.linker import DynamoLinker, RedisLinker, SingletonCreationError
 
-TEST_IDS = ['test_id_1']
+TEST_IDS = [uuid4()]
```

Comment:
you need to test more cases. 
Like these.
Input : uuid => search as uuid
Input: uuid => search as str
Input: str => search as str
Input: str => search as uuid